### ISSUE TITLE
[macOS] Tooltip doesn't resize

### ISSUE THREAD

**Issue by ErinaceusLinnaeus** [momentarylapse/tsunami#23]
When hovering over a button in the create-new-track window, sometimes the tooltip doesn't get resized.

Reproduction:
Here I hovered over Midi and went straight right. Metronome, Master and Preset used the same size as Midi:
![2024-11-11 12 29 59](https://github.com/user-attachments/assets/85b96313-87c4-4e82-b88b-995ec04f8a6e)
![2024-11-11 12 30 11](https://github.com/user-attachments/assets/1cf5ae98-d7f9-4f61-88cd-7040a38c6ae8)
![2024-11-11 12 30 16](https://github.com/user-attachments/assets/75e7953c-e92b-4f21-b478-e441b4ee9480)

Here I hovered over Audio and then went right:
![2024-11-11 12 30 51](https://github.com/user-attachments/assets/4c6b4003-b3f9-4858-b557-19010d63ab39)

Reseting this behaviour by movintg the mouse up or down. Coming to another button not directly from another button that showed its tooltip.

**Comment by momentarylapse** [comment]
Yes, I can confirm this bug also happens for me on MacOS. But it does not seem to happen on linux.

I'm afraid the rendering of those tooltips is completely handled by `gtk4`. Might be a bug in the library, although I could not find anything online. Or maybe I messed something up. I'll keep an eye out, but no promises.
